libs/blocks/media/media.css

```diff
@@ -341,8 +342,9 @@ div[class*="-up"] .media .foreground > .media-row {
 
   .media.qr-code img.qr-code-img {
```

Comment:
Is the added tag specificity actually needed? Would `.media.qr-code .qr-code-img` not achieve the same result?

Reply:
I believe it will. However as it was not required to update for the changes requested in the ticket, I didn't touched it. I can follow up with original contributor to check the reason and may be address this in a separate PR.